src/UI/Art/SpriteArtist/InfoView.java

```diff
@@ -92,7 +92,12 @@ public void render(Graphics g)
       return; // Continue no further if we don't have room to actually draw
 
     if (null != thisArmy)
-      CommanderOverlayArtist.drawCommanderOverlay(myG, thisArmy, false);
+    {
+      MapPerspective drawableMap = getDrawableMap(myControl.getGame());
+      boolean fogOn = myControl.getGame().isFogEnabled();
+      boolean hideFunds = fogOn && thisArmy.isEnemy(drawableMap.viewer);
+      CommanderOverlayArtist.drawCommanderOverlay(myG, hideFunds, thisArmy, false);
```

Comment:
Intent here is to show the enemy Commander's unit count/income/etc based on what the active CO can see, right? That feels potentially misleading, since there is no disclaimer. Is this what the carts do?
Also it looks like the vehicle count is counting troops instead of vehicles (see COStateInfo line 46).

Reply:
Yes, it's been that way since we've had fog.
It's a little weird, sure, but it's not hard to figure out how it works... and it saves you calculation without giving you free info.
The carts (weirdly) give you a complete unit count (and units lost), but nothing on funds. I guess that makes sense when rout is a major objective and the game's focused on single player. They also give you a complete property count if capture limit is on.
> Also it looks like the vehicle count is counting troops instead of vehicles (see COStateInfo line 46).

You appear to be correct.